python/lsst/daf/butler/registry/managers.py

```diff
@@ -229,6 +230,11 @@ class RegistryManagerInstances(
     `Registry`.
     """
 
+    column_types: ColumnTypeInfo
+    """Information about column types that can differ between data repositories
+    and registry instances, including the dimension universe.
+    """
+
```

Comment:
FYI, following some initial work on [DM-36326](https://jira.lsstcorp.org/browse/DM-36326), I'm planning to pare this commit back significantly to just adding the `column_types` attribute here without passing it to all of the manager `initialize` methods.  I'm now pretty confident I can avoid the chain of events that made me need it on downstream branches, and if I do need it after all I'll bring back that part of the commit on a later ticket.  I'm not planning to rebase this branch until you're done with your review, though, unless you'd like me to (I have the updated branch in hand, just not pushed).

Reply:
Push your updated branch. I had started looking but hadn't made much progress because of JSR distractions.